### DISCUSSION TITLE
Script as input file?

### DISCUSSION THREAD

**Discussion by sbegg** [mulesoft-labs/data-weave-native#61] [Ideas]
I regularly use the DW playground but it cannot handle large input.
I regularly use Studio to test DW scripts but with large input sit can do a JVM ping (death) timeout.
I don't want to deploy my Mule app to Cloudhub and have to debug logs, so I would like  a local DW engine that I can feed a complex script, which seems not possible with dw - i input {source-dat-filea < some-big-scripty-file.
Did I miss something.
Thanks.

**Comment by machaval** [comment]
The way to use the cli would be 
```bash
> dw -i <inputName> <pathToBigInput> -f <pathToDWFile>
```
For example
```bash
> dw -i  payload ./MyBigJson.json -f ./MyDWScript.dwl
```

**Comment by sbegg** [comment]
As I understood from the README, -f, --file means

Specifies output file for the transformation.

**Reply by machaval** [reply]
No to specify the output is `-o` . From the --help you will get that

```
 -f,--file <file-path>                     Specifies the DataWeave file
                                           path to execute.
 -o,--output <output-path>                 Specifies output file for the
                                           transformation if not standard
                                           output will be used.
```

So if you want to specify the output to be a file instead of the standard output. You can use for example `-o ./MyOutputFile.json`

**Comment by sbegg** [comment]
Yes!
Thanks for clarifying. I must have read an old version of the README.
